docs/quickstarts/outlook-quickstart-json-manifest.md

```diff
@@ -0,0 +1,271 @@
+
+1. Run the following command in the root directory of your project. When you run this command, the local web server starts and your add-in will be [sideloaded](../outlook/sideload-outlook-add-ins-for-testing.md). 
+
+    ```command&nbsp;line
+    npm start
+    ```
+
+1. Use the Classic Ribbon in Outlook. The remainder of these instructions assume this.  
+
+1. View a message in the [Reading Pane](https://support.microsoft.com/office/2fd687ed-7fc4-4ae3-8eab-9f9b8c6d53f0), or open the message in its own window. A new control group named **Contoso Add-in** appears on the Outlook **Home** tab (or the **Message** tab if you opened the message in a new window). The group has a button named **Show Taskpane** and one named **Perform an action**.
+
+    > [!NOTE]
+    > If the new group isn't present, then your add-in wasn't automatically sideloaded. Follow the instructions in [Sideload manually - Outlook 2016 or later on Windows or Mac](../outlook/sideload-outlook-add-ins-for-testing.md#outlook-2016-or-later-on-windows-or-mac) to manually sideload the add-in in Outlook. When you are prompted to upload the manifest file, use the file `C:\Users\{your_user_name}\AppData\Local\Temp\manifest.xml`. The file has an `.xml` extension because during the preview period, the JSON-formatted manifest is converted to an XML manifest, which is then sideloaded.
+
+1. Select the **Perform an action** button. It [executes a command](../develop/create-addin-commands.md?branch=outlook-json-manifest#step-5-add-the-functionfile-element) to generate a small informational notification at the bottom of the message header, just above the message body.
+
+1. When prompted with the **WebView Stop On Load** dialog box, select **OK**.
+
+    [!INCLUDE [Cancelling the WebView Stop On Load dialog box](../includes/webview-stop-on-load-cancel-dialog.md)]
+
+1. Choose the **Show Taskpane** button in the ribbon to open the add-in task pane.
+
+    > [!NOTE]
+    > If you receive the error "We can't open this add-in from localhost" in the task pane, follow the steps outlined in the [troubleshooting article](/office/troubleshoot/office-suite-issues/cannot-open-add-in-from-localhost).
+
+1. When prompted with the **WebView Stop On Load** dialog box, select **OK**.
+
+    [!INCLUDE [Cancelling the WebView Stop On Load dialog box](../includes/webview-stop-on-load-cancel-dialog.md)]
+
+1. Scroll to the bottom of the task pane and choose the **Run** link to copy the message's subject to the task pane.
+
+1. End the debugging session with the following command:
+
+    ```command&nbsp;line
+    npm stop
+    ```
+
+    > [!IMPORTANT]
+    > Closing the web server window does not reliably shut down the web server. If it is not properly shut down, you'll encounter problems as you change and rerun the project.
+
+1. Close all instances of Outlook.
+
+## Add a custom button to the ribbon
+
+Let's add a custom button to the ribbon that inserts text into a message body.
```

Comment:
```suggestion
Add a custom button to the ribbon that inserts text into a message body.
```
We never use first-person plural tense. It is the opposite of Microsoft's modern voice. The exception is "we recommend", but even that is used sparingly. Use second person as if you're talking to the person. The "you" is implied, which is why we don't use "you can."